index.bs

```diff
@@ -3550,9 +3492,9 @@ the following algorithm returns <i>true</i>.
         </pre>
 
         <code>CBIface</code> is distinguishable from <code>Iface</code>
-        because the pair satisfies note (a),
+        because there's a ● at the intersection of dictionary-like and interface-like,
```

Comment:
I realized today that I also had to update this example now that we aren't using the notes to handle callback interfaces.